Detections/SecurityEvent/UserAccountCreatedDeleted_10m.yaml

```diff
@@ -25,17 +25,17 @@ query: |
   let timeframe = 1d;
   let spanoftime = 10m;
   let threshold = 0;
-  SecurityEvent 
-  | where TimeGenerated > ago(2*timeframe) 
+  SecurityEvent
+  | where TimeGenerated > ago(timeframe+spanoftime)
```

Comment:
Agreed, the old value doesn't actually help because the comparison time frame is only 1d. :)  Adding in the extra 10m just provides the ability to check for the absolute full match and based on timings could help pick up a miss on a daily run.